app/api/services/users.py

```diff
@@ -0,0 +1,41 @@
+from app.api.helpers import Service
+from app import db
+from app.models import User
+from flask_login import current_user
+from sqlalchemy import func
+from sqlalchemy.sql.functions import concat
+
+
+class UsersService(Service):
+    __model__ = User
+
+    def __init__(self, *args, **kwargs):
+        super(UsersService, self).__init__(*args, **kwargs)
+
+    def get_email_domain(self):
+        """Returns the user's email domain without the '@' symbol."""
+        return current_user.email_address.split('@')[-1]
```

Comment:
I think the intention of the services classes is for DB interactions.  Meaning, there shouldn't be anything with Flask here.
I think the correct place for this would be app.api.helpers